lib/twilio/wstransport.ts

```diff
@@ -515,6 +537,34 @@ export default class WSTransport extends EventEmitter {
     this.state = state;
   }
 
+  /**
+   * Perform a backoff based on the signaling retryAfter value.
+   */
+  private _retryAfterBackoff(retryAfter: number): void {
```

Comment:
How is `retryAfter` being used in this method? I only see it logged and doesn't actually control how long to wait before reconnecting.